osmdroid-android/build.gradle

```diff
@@ -2,6 +2,10 @@ description = 'OSMdroid Android'
 apply plugin: 'com.android.library'
 apply from: 'https://raw.githubusercontent.com/chrisdoyle/gradle-fury/master/gradle/android-support.gradle'
 
+configurations {
```

Comment:
not sure why this is here, you can generate javadocs with
gradlew install -Pprofile=javadoc